src/lang/en.json

```diff
@@ -799,5 +799,6 @@
     "nostrRecipients": "Recipients Public Keys (npub)",
     "nostrRecipientsHelp": "npub format, one per line",
     "showCertificateExpiry": "Show Certificate Expiry",
-    "noOrBadCertificate": "No/Bad Certificate"
+    "noOrBadCertificate": "No/Bad Certificate",
+    "cacheBusterParam": "Add Cache Buster Param"
```

Comment:
What do you think about either of the following suggestions with adding `<code>uptime_kuma_cachebuster</code>` as a i18n-component template:

```suggestion
    "cacheBusterParam": "Add {0} as a parameter with random content ('cache buster')"
```

```suggestion
    "cacheBusterParam": "Add {0} as a cache buster parameter"
```

Reply:
I think it makes a lot of sense to make it clear what he param is called here, are we happy with the text wrapping here?   
![image](https://github.com/louislam/uptime-kuma/assets/372067/d2646eee-0346-4a19-a41f-4d6373acd0fc)

Reply:
thinking about it, I think we can even drop that it is a cache buster, given that that is literally written in the parameter name. Thoughts?

Reply:
Thats true 
![image](https://github.com/louislam/uptime-kuma/assets/372067/4489d789-4cc4-4c95-bf0e-b5e3802e0654)

Reply:
Could you utilise the `i18n-t`-component to make this into `<code>uptime_kuma_cachebuster</code>`?

Should we add a helptext below it, or is cache busting a common enough thing that average users know what that means

Reply:
Apologies I missed your original meaning about the i18n component.  How about this: 
![image](https://github.com/louislam/uptime-kuma/assets/372067/61b6d0a9-a05a-47fa-875b-766583ffc977)